### PR TITLE
[DE-438] feat/add tests on primary and foreign keys on global tables

### PR DESCRIPTION
**Pull request by LucileRainteau**
# DE PR

## Describe your changes

Please include a summary of the changes:

This PR adds tests on primary and foreign keys to 3 tables : global_offer, global_booking and global_venue.
The commented tests on the yaml are disfunctional tests in prod.

Tag a reviewer if necessacy  @github/username 

## Jira ticket number and/or notion link

JIRA-ticket_number

### Type of change
- [ ] Fix (non-breaking change which corrects expected behavior)
- [ ] New fields (non-breaking change)
- [ ] New table (non-breaking change)
- [ ] Concept change (potentially breaking change which modifies fields according to new or evolving business concepts) 
- [ ] Table deletion (potentially breaking change which adds functionality/ table)
      
### Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] My code passes CI/CD tests
- [ ] I updated README.md
- [ ] I have updated the dag
- [ ] If my changes concern incremental table, I have altered their schema to accomodate with field's creation/deletion
- [ ] I have made corresponding changes to the [tables documentation](https://www.notion.so/passcultureapp/Documentation-Tables-175a397a8e854ff4a55ae4f3620dbe3b)
- [ ] I have made corresponding changes to the [fields glossary](https://www.notion.so/passcultureapp/854a436a8f1541e1b6ec2a65f8bab600?v=798024ba90404b139e5a17407a3bc604)
- [ ] I will create a review on slack and ensure to specify the duration of the review task: short (<10min), medium (<30min), long (>30min)

### Added tests?
- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help
- [ ] ⏰ no, but I created a ticket

### CODE REVIEW

File: orchestration/dags/data_gcp_dbt/models/mart/global/schema/mrt_global__booking.yml [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+        description: ''
+        data_type: DATETIME
+      - name: booking_cancellation_reason
+        description: ''
+        data_type: STRING
+      - name: user_id
+        description: ''
+        data_type: STRING
+        tests:
+          - not_null
```
Comment (carolinemestre-passculture) [inline orchestration/dags/data_gcp_dbt/models/mart/global/schema/mrt_global__booking.yml:42]: Comme on a un `INNER JOIN` avec la table user dans le modèle, je pense pas que ce test soit utile car par définition du modèle il peut pas être `null` ce champs

File: orchestration/dags/data_gcp_dbt/models/mart/global/schema/mrt_global__booking.yml [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+        description: ''
+        data_type: DATETIME
+      - name: stock_beginning_date
+        description: ''
+        data_type: DATETIME
+      - name: stock_id
+        description: ''
+        data_type: STRING
+        tests:
+          - not_null
```
Comment (carolinemestre-passculture) [inline orchestration/dags/data_gcp_dbt/models/mart/global/schema/mrt_global__booking.yml:68]: Idem pour le `stock_id`

File: orchestration/dags/data_gcp_dbt/models/mart/global/schema/mrt_global__booking.yml [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+      - name: offer_id
+        description: ''
+        data_type: STRING
+        tests:
+          - not_null
```
Comment (carolinemestre-passculture) [inline orchestration/dags/data_gcp_dbt/models/mart/global/schema/mrt_global__booking.yml:73]: Pareil ici on a un filtre dans le modèle `AND s.offer_id IS NOT NULL` donc pas sûr que ce test soit nécessaire

File: orchestration/dags/data_gcp_dbt/models/mart/global/schema/mrt_global__offer.yml [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+        description: ''
+        data_type: INT64
+      - name: total_first_bookings
+        description: ''
+        data_type: INT64
+      - name: venue_id
+        description: ''
+        data_type: STRING
+        tests:
+          - not_null
```
Comment (carolinemestre-passculture) [inline orchestration/dags/data_gcp_dbt/models/mart/global/schema/mrt_global__offer.yml:184]: Comme on a le filtre `AND venue_id IS NOT NULL` on peut le retirer

**Review by carolinemestre-passculture — COMMENTED** [review]
J'ai mis des commentaires sur des tests qui sont redondants avec les modèles que l'on pourrait retirer. A nous de voir si on préfère les laisser quand même (pour par exemple si jamais quelqu'un retire un filtre ou update un join, que le test fail).

**Review by valoumiaou — APPROVED** [review]

**Comment by valoumiaou** [comment]
test failing normal

**Pull request merged**